ui/imports/shared/popups/ImportCommunityPopup.qml

```diff
@@ -49,85 +60,136 @@ StatusDialog {
+                wrapMode: Text.WordWrap
                 font.pixelSize: 13
-                visible: keyInput.text.trim() !== ""
-                text: {
-                    if (d.errorMessage !== "") {
-                        return d.errorMessage
-                    }
-                    if (d.isPrivateKey) {
-                        return qsTr("Private key detected")
+                color: Theme.palette.baseColor1
+            }
+
+            StatusBaseText {
+                id: inputLabel
+                text: qsTr("Community key")
+                color: Theme.palette.directColor1
+                font.pixelSize: 15
+            }
+
+            StatusTextArea {
+                id: keyInput
+                Layout.fillWidth: true
+                implicitHeight: 108
+                placeholderText: "0x0..."
+                wrapMode: TextEdit.WrapAtWordBoundaryOrAnywhere
+                onTextChanged: d.importErrorMessage = ""
+            }
+            RowLayout {
+                Layout.fillWidth: true
+                Layout.minimumHeight: 46
+                Layout.maximumHeight: 46
+                StatusChatInfoButton {
+                    visible: d.communityFound
+                    title: visible ? d.communityDetails.name : ""
+                    subTitle: visible ? qsTr("%n member(s)", "", d.communityDetails.nbMembers) : ""
+                    asset.name: visible ? d.communityDetails.image : ""
+                    asset.isImage: (asset.name !== "")
+                    asset.color: visible ? d.communityDetails.color : ""
+                }
+                Item { Layout.fillWidth: true }
+                StatusBaseText {
+                    id: detectionLabel
+                    Layout.alignment: Qt.AlignRight
+                    font.pixelSize: 13
+                    visible: keyInput.text.trim() !== ""
+                    text: {
+                        if (d.errorMessage !== "") {
+                            return d.errorMessage
+                        }
+                        if (d.isPrivateKey) {
+                            return qsTr("Private key detected")
+                        }
+                        if (d.isPublicKey) {
+                            return qsTr("Public key detected")
+                        }
                     }
-                    if (d.isPublicKey) {
-                        return qsTr("Public key detected")
+                    color: d.errorMessage === "" ? Theme.palette.successColor1 : Theme.palette.dangerColor1
+                }
+            }
+            ColumnLayout {
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+                visible: (d.communityFound && d.isPrivateKey)
+                Layout.topMargin: 12
+                spacing: Style.current.padding
+                Rectangle   {
```

Comment:
1. Not sure if I'm looking at the correct [figma design](https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?type=design&node-id=16038-386170&mode=design&t=AOAtjoDcqwI9gm1D-0), if yes, this seems to have a background color and some more padding on top and bottom. 
Design:
<img width="926" alt="Screenshot 2023-08-04 at 09 45 14" src="https://github.com/status-im/status-desktop/assets/97019400/08006d5b-5337-4eb2-98ed-dcd3a63e274e">

2. Suggestion: I've seen this warning box in another design and will need to be added in another context. Maybe it would worth it to create a separate component for it. Here, [the other usage.] (https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?type=design&node-id=35910-617973&mode=design&t=02nMBLErk543Ce2l-0)